src/main/java/lotes/parser/Parser.java

```diff
@@ -0,0 +1,107 @@
+package lotes.parser;
+
+import java.util.Scanner;
+
+import lotes.storage.Storage;
+import lotes.task.TaskList;
+
+public class Parser {
```

Comment:
Parser is doing both parsing and executing the command. in iP this is kinda OK. However, you are violating some design principles you will learn about soon. Avoid this in tP